draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -239,6 +273,46 @@
             staying within the maximum character transmission rate (<xref target="sec.sdp.dcsa"/>),
             if such has been provided by the peer.
           </t>
+          <t>
+            An implementation needs to take the user requirements for smooth flow and low
+            latency in real-time text conversation into consideration when assigning a buffer time.
+            It is RECOMMENDED to use the default transmission interval of 300 milliseconds
+            <xref target="RFC4103"/>, or lower, also for T.140 data channels.
+          </t>
+        </section>
+        <section anchor="sec.t140.loss" title="Loss of T140blocks">
+          <t>
+            In case of network failure or congestion, T140 data channels might get lost.
+            If this happens but the session sustains, it is RECOMMENDED that a low number of 
+            retries are made to reestablish the T140 data channels. If reestablishment of
+            the T140 data channel is successful, an implementation MUST evaluate if any 
+            T140blocks were lost. Retransmission of already transmitted T140blocks MUST be
+            avoided, and missing text markers <xref target="T140ad1"/> SHOULD be inserted in the received 
+            data stream where loss is detected or suspected and presented to the user.
+          </t>
+          <t>
+            An implementation needs to take the user requirements for smooth flow and low
+            latency in real-time text conversation into consideration when assigning a buffer time.
+            It is RECOMMENDED to use the default transmission interval of 300 milliseconds
+            <xref target="RFC4103"/>, or lower, also for T.140 data channels.
+          </t>
+        </section>
+        <section anchor="sec.t140.mul" title="Multi-party Considerations">
+          <t>
+            If an implementation needs to support multi-party scenarios, implementations need
+            to support multiple simultaneous T.140 data channels, one for each remote party. At
+            the time of writing this document, this is true even in scenarios where each participants
+            communicate via a centralized conference server. The reason is that, unlike RTP media,
+            WebRTC data channels and the T.140 protocol do not support the indication of the source
+            of T.140 data. The SDP dcmap attribute label attribute parameter (<xref target="sec.sdp.dcmap"/>
```

Comment:
do we need to say "SDP dcmap attribute label attribute parameter"? It gets complex.
Can it do with "dcmap label attribute" ?

Reply:
"Attribute" refers to an SDP attribute, and label is not an attribute. We have used this type of wording also in other documents, and I think we should use the complete name at least on first occurrence. I will see if it can be simplified elsewhere.

(Note that there is also an SDP label attribute, defined in RFC 4574, but that is not what we are talking about in the draft, and that needs to be clear in the text.)